src/LinkDotNet.NCronJob/Scheduler/QueueWorker.cs

```diff
@@ -56,17 +60,30 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
         try
         {
-            while (!stopToken.IsCancellationRequested && jobQueue.Count > 0)
+            while (!stopToken.IsCancellationRequested)
```

Comment:
We have to remove the `jobQueue.Count > 0` as there might be 0 CRON jobs, but a user wants to trigger instant jobs.